AGENTS.md

```
@@ -0,0 +1,74 @@
+Instructions for AI coding agents working on Rein. Read this before making changes.
```

markdownlint-cli2 (0.23.2)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)